docs/src/components.js

```diff
@@ -153,6 +169,20 @@ const components = {
   ShadeDemo,
   ShadeOtherDemo,
   TextDemo,
+  Attention,
+  Bookmark,
+  Calendar,
+  Chat,
+  Check,
+  Email,
+  Flame,
+  Information,
+  Key,
+  Pin,
+  Plus,
+  Search,
+  TrendingUp,
+  Warning,
```

Comment:
We might also want to add a comment that only icons exported here are available for use in markdown

Reply:
Good idea, added!